sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -439,55 +439,83 @@ Text Analytics for health is a containerized service that extracts and labels re
 
     string document2 = "Prescribed 100mg ibuprofen, taken twice daily.";
 
-    List<string> batchInput = new List<string>()
```

Comment:
to keep the readme simple, it will be better if we use the convenience method (which is easier to use) instead of this one.